### PR TITLE
adding Simulink interface

### PR DESCRIPTION
**Pull request by zanellia**
This PR adds a simple Simulink interface. Additional features will be added upon request. @giaf: let's add some code to copy around the S-Function when doing codegen?

### CODE REVIEW

**Comment by zanellia** [comment]
ready to merge, I think

File: examples/simulink/qp_data.c [hunk 1]
```diff
@@ -0,0 +1,682 @@
+/***************
```
Comment (giaf) [inline examples/simulink/qp_data.c]: What about using the 'getting_started' QP example? This mass-spring data file is quite big

**Comment by zanellia** [comment]
Somehow this was heavily deprioritized... I have now added a very simple function call to the MEX getters to fetch the dimensions automatically.

**Comment by zanellia** [comment]
I guess we can merge this!

**Comment by giaf** [comment]
ok!

**Pull request merged**